### ISSUE TITLE
gpaste 3.20 undefined reference to symbol 'gtk_clipboard_wait_for_text'

### ISSUE THREAD

**Issue by kapouer** [Keruspe/GPaste#143]
However i'm pretty certain i have gtk-3 >= 3.20 here on debian/expertimental/sid
```
libtool: link: gcc -pthread -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -pthread -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng12 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wall -pipe -pedantic -DANOTHER_BRICK_IN_THE -Wall -W -Wextra -Wvla -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing=2 -Wwrite-strings -Wno-unknown-warning-option -Werror=overflow -Wp,-D_FORTIFY_SOURCE=2 -ffast-math -fno-common -fdiagnostics-show-option -fno-strict-aliasing -fvisibility=hidden -ffunction-sections -fdata-sections -Wl,-z -Wl,relro -Wl,--as-needed -Wl,--gc-sections -o bin/.libs/clipboard-watcher src/clipboard-watcher/bin_clipboard_watcher-clipboard-watcher.o  ./lib/.libs/libgpaste.so -pthread
/usr/bin/ld: src/clipboard-watcher/bin_clipboard_watcher-clipboard-watcher.o: undefined reference to symbol 'gtk_clipboard_wait_for_text'
//usr/lib/x86_64-linux-gnu/libgtk-3.so.0: error adding symbols: DSO missing from command line
```

**Comment by Keruspe** [comment]
Should be fixed by 3503fcabcd48c53f5d9ba94ecd94569db6323b63

**Comment by kapouer** [comment]
It works, thank you.
